src/components/quick-settings.tsx

```diff
@@ -0,0 +1,48 @@
+import { TranslationBundle } from '@jupyterlab/translation';
+import type { CommandRegistry } from '@lumino/commands';
+import { MenuSvg, settingsIcon } from '@jupyterlab/ui-components';
+import * as React from 'react';
+
+import { CommandIDs } from '../types';
+
+export function QuickSettings(props: {
+  trans: TranslationBundle;
+  commands: CommandRegistry;
+}): React.ReactElement {
+  const { commands } = props;
+
+  const menu = new MenuSvg({ commands: commands });
+  menu.addItem({ command: CommandIDs.showStarred, args: {} });
+  menu.addItem({ command: CommandIDs.searchAllSections, args: {} });
+  menu.addItem({ command: CommandIDs.openSettings, args: {} });
```

Comment:
Nitpick: I think that `args` are optional. Unless there is a reason to use them?

```suggestion
  menu.addItem({ command: CommandIDs.showStarred });
  menu.addItem({ command: CommandIDs.searchAllSections });
  menu.addItem({ command: CommandIDs.openSettings });
```